editor/editor.tsx

```diff
@@ -1381,84 +1388,14 @@ function Editor(props: RouteComponentProps) {
-                    }}
-                >
-                    <h1>Gosling.js Examples</h1>
-                    {ExampleGroups.filter(_ => _.name !== 'Doc' && _.name !== 'Unassigned').map(group => {
-                        return (
-                            <>
-                                <h2 id={`${group.name}`}>{group.name}</h2>
-                                <h5>{group.description}</h5>
-                                <div className="example-group" key={group.name}>
-                                    {Object.entries(examples)
-                                        .filter(d => !d[1].hidden)
-                                        .filter(d => d[1].group === group.name)
-                                        .map(d => {
-                                            return (
-                                                <button
-                                                    id={`${d[1].group}_${d[1].name}`}
-                                                    title={d[1].name}
-                                                    key={d[0]}
-                                                    className="example-card"
-                                                    onClick={() => {
-                                                        setShowExamples(false);
-                                                        closeDescription();
-                                                        setIsImportDemo(true);
-                                                        setDemo({ id: d[0], ...examples[d[0]] } as any);
-                                                    }}
-                                                >
-                                                    <div
-                                                        className="example-card-bg"
-                                                        style={{
-                                                            backgroundImage: d[1].image ? `url(${d[1].image})` : 'none'
-                                                        }}
-                                                    />
-                                                    <div className="example-card-name">{d[1].name}</div>
-                                                </button>
-                                            );
-                                        })}
-                                </div>
-                            </>
-                        );
-                    })}
-                    {/* Just an margin on the bottom */}
-                    <div style={{ height: '40px' }}></div>
-                </div>
-            </div>
+            {showExamples && (
+                <EditorExamples
+                    setShowExamples={setShowExamples}
+                    closeDescription={closeDescription}
+                    setIsImportDemo={setIsImportDemo}
+                    setDemo={setDemo}
+                />
+            )}
```

Comment:
Pull out the editor examples into its own component. The conditional rendering is much more clear this way

Reply:
Thanks for making this change!